opengrok-indexer/src/test/java/org/opengrok/indexer/index/IndexerMainTest.java

```diff
@@ -34,7 +34,7 @@
 
 import static org.junit.Assert.assertFalse;
 
-public class IndexerMainTest {
+public class  IndexerMainTest {
```

Comment:
Why the additional space? Maybe a mistake?

Reply:
yep, inadvertent modification.